go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/go-git/go-billy/v5 v5.5.0
 	github.com/google/go-containerregistry v0.19.2
 	github.com/google/go-github/v61 v61.0.0
+	github.com/moby/buildkit v0.14.0
```

Comment:
Yes, this is a new dependency, but looking at the indirect deps and go.sum, the added weight doesn't seem to be so bad. And we might need this to actually take the platform string into use.

Reply:
That's fine. This seems like a well-known library (trusty score 7.6) and it reduces the amount of code we have to write.